stix_shifter_modules/azure_log_analytics/stix_translation/json/SecurityEvent_from_stix_map.json

```diff
@@ -0,0 +1,171 @@
+{
+    "ipv4-addr": {
+        "fields": {
+            "value": [
+                "IpAddress"
+            ]
+        }
+    },
+    "domain-name": {
+        "fields": {
+            "value": [
+                "TargetDomainName"
+            ]
+        }
+    },
+    "user-account": {
+        "fields": {
+            "account_login": [
+                "TargetUserName"
+            ]
+        }
+    },
+    "directory": {
+        "fields": {
+            "path": [
+                "HomeDirectory",
+                "HomePath"
+            ]
+        }
+    },
+    "file": {
+        "fields": {
+            "path": [
+                "FilePath"
+            ],
+            "hashes": [
```

Comment:
I don't think there would every be a search for just `hashes`, would that even be a valid property or wold it always be `hashes.'SHA-1'` `hashes.'MD5'` and so on?

Reply:
ohh didn't notice they put this under hashes. the property is simply called FileHash so all types of hashes will basically be translated into `FileHash` filed in the query.

Reply:
And we  need to do some post processing before results translation to correctly map the hash type in the bundle